Exec/radiation_tests/RadSuOlsonMG/probin.common

```diff
@@ -16,4 +16,14 @@
   eos_c_v_exp_m = 0.0d0
   eos_c_v_exp_n = -3.0d0
 
+  const_kappa_r = 1.9801980198019803d-31
```

Comment:
this was not set originally

Reply:
That's right, but I'm preserving the previous behavior, which was that the code was forcing kappa_r to be equal to kappa_r if kappa_p was set but kappa_r was not.